### PR TITLE
T/qa 2157/apply black

### PR DESCRIPTION
**Pull request by uc-pperumala**
## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Review by bpierce01 — CHANGES_REQUESTED** [review]
There are merge conflicts

**Comment by uc-pperumala** [comment]
> There are merge conflicts

@bpierce01 fixed. should be good now.

**Review by bpierce01 — CHANGES_REQUESTED** [review]
Getting a compile error when trying to run: 
Failed to parse /gdc_data_portal_v2/pages/cohort_bar.py: Error, got an unexpected token EQUAL here:

  93         """
  94         # Concatenate the full text of the filter being checked for
  95         full_query_filter = filter + values
  96         full_query_filter_locator = CohortBarLocators.TEXT_COHORT_QUERY_FILTER(            
  97 full_query_filter, position        
  98 )
  99         
 100         = <---- here

File: holmes-py/step_impl/apps/gdc_data_portal_v2/pages/cohort_bar.py [hunk 1]
```diff
@@ -69,15 +92,26 @@ def is_cohort_query_filter_present(self, filter, values, position):
         :return: True or False is the filter present
         """
         # Concatenate the full text of the filter being checked for
-        full_query_filter = filter+values
-        full_query_filter_locator = CohortBarLocators.TEXT_COHORT_QUERY_FILTER(full_query_filter, position)
-        is_full_query_filter_locator_visible = self.is_visible(full_query_filter_locator)
```
Comment (bpierce01) [inline holmes-py/step_impl/apps/gdc_data_portal_v2/pages/cohort_bar.py]: This variable "is_full_query_filter_locator_visible" was erroneously removed

**Review by bpierce01 — APPROVED** [review]
Ran regression, did not find any problems. Code formatting looks fine. Approving

**Comment by uc-pperumala** [comment]
@craigrbarnes could you please merge this whenever you have a chance?

**Pull request merged**